ferret/explainers/explanation_speech/equal_width/loo_equal_width_explainer.py

```diff
@@ -65,19 +63,17 @@ def compute_explanation(
         Computes the importance of each equal width audio segment in the audio.
         """
 
-        ## Load audio as pydub.AudioSegment
-        audio = AudioSegment.from_wav(audio_path)
-        audio_np = pydub_to_np(audio)[0]
+        audio_array = audio.array
 
         ## Remove word
         audio_remove_segments = []
 
-        duration_s = len(audio) / 1000
+        duration_s = len(audio_array) / audio.sample_rate # finds the duration from the array 
```

Comment:
Same double-check as in `gradient_equal_width_explainer.py`.

Reply:
the denominator `1000` assumed to work with milliseconds. Is this change still valid? @emanuele-moscato